package.json

```diff
@@ -27,14 +27,14 @@
     "async": "^3.2.0",
     "benchmark": "^2.1.4",
     "fastify": "^4.0.1",
-    "fastify-postgres": "^3.1.0",
     "jsonfile": "^6.1.0",
     "napa": "^3.0.0",
     "pg": "^8.6.0",
     "sql-template-strings": "^2.2.2",
     "standard": "^17.0.0",
     "tap": "^16.0.0",
-    "tsd": "^0.25.0"
+    "tsd": "^0.25.0",
+    "@fastify/postgres": "^4.0.0"
```

Comment:
use the package manager to install this, I don't think it would be positioned here if you used npm